src/components/molecules/Cv/components/Skills.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react';
+import { Box, Text, BoxProps } from '../../../quarks';
+import { CvHeading, Timeline } from '../../../atoms';
+
+export const Skills: FC<ISkillsProps> = ({ skills, heading, educationDescription }) => (
+  <Box>
+    <CvHeading heading={heading} />
+    {educationDescription}
+    <Timeline>
+      {typeof skills === 'object' ? (
```

Comment:
nie rozumiem czemu tak?

Reply:
mamy przypadek gdzie przekazujemy po prostu string? wtedy nazwa skills jest myląco bo to jeden skill :D

Reply:
ponieważ w przypadku braku doświadczenia chcę wyświetlić tylko string z informacją 'Szuka pierwszej pracy' ;>

Reply:
w takim wypadku jakakolwiek nazwa będzie myląca ;) zaproponuj coś albo zostawiam